### PR TITLE
release: v1.0.0a14

### PR DESCRIPTION
**Pull request by hachreak**
Signed-off-by: Leonardo Rossi <leonardo.r@cern.ch>

### CODE REVIEW

**Comment by lnielsen** [comment]
Please remove the post release version bump commit.

**Comment by hachreak** [comment]
@lnielsen removed

**Comment by hachreak** [comment]
@lnielsen could you please release the new version? I need for another PR. thanks :)

File: RELEASE-NOTES.rst [hunk 1]
```diff
@@ -1,8 +1,8 @@
 =========================
- Invenio Theme v1.0.0a13
+ Invenio Theme v1.0.0a14
 =========================
 
-Invenio Theme v1.0.0a13 was released on June 15, 2016.
+Invenio Theme v1.0.0a14 was released on June 15, 2016.
```
Comment (lnielsen) [inline RELEASE-NOTES.rst]: June 15 -> August 25

**Pull request closed without merging**